CONTRIBUTING.md

```diff
@@ -220,14 +256,37 @@ that, you want to use the web extension profile in the launch setup.
 The default build target will allow you to debug the extension by providing the
 right sourcemaps.
 
+## Releasing
+
+`coq-lsp` is released using `dune-release tag` + `dune-release`.
+
+The checklist for the release as of today is the following:
+
+### Client:
+
+- update the client changelog at `editor/code/CHANGELOG.md`, commit
```

Comment:
We could be a bit more specific here and note "update the client changelog at `editor/code/CHANGELOG.md` from `CHANGES.md` at the root